src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -604,6 +629,25 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Viewer: {
+        commerceOrders: {
```

Comment:
Then extend `Viewer` via schema delegation, and point it at the `commerceOrders` field.

(It is impossible to expect anyone anywhere to understand this 😭)

Reply:
🙏🏻 🙏🏻 
Bless your kind soul

Reply:
😅 STITCHING 😓